6_visualize/src/prep_legend_fun.R

```diff
@@ -41,6 +42,8 @@ prep_legend_fun <- function(precip_bins, legend_styles, timesteps_ind, storm_poi
     ybottom <- coord_space[3]
     dot_x <- x_edge+bin_w/2*shift_dir
     dot_txt_x <- x_edge+bin_w*0.7*shift_dir
+    seg_x <- x_edge+bin_w/3*shift_dir
+    center_to_txt_y <- strheight("A")/3 # height of character divided by three seems to do the trick
```

Comment:
agreed - this is a bit magical, especially since the text elements below add a cex=1.5 that would make strheight not exactly accurate, but it does look good, so let's keep it until the time if and when we need something more directly tied to text height (which may never happen)